machine_test.go

```diff
@@ -1939,7 +1939,19 @@ func TestLoadSnapshot(t *testing.T) {
 			},
 
 			loadSnapshot: func(ctx context.Context, machineLogger *logrus.Logger, socketPath, memPath, snapPath string) {
-				cfg := createValidConfig(t, socketPath+".load")
+				// Note that many fields are not necessary when loading a snapshot
+				cfg := Config{
+					SocketPath: socketPath + ".load",
+					Drives: []models.Drive{
+						{
+							DriveID:      String("root"),
+							IsRootDevice: Bool(true),
+							IsReadOnly:   Bool(true),
+							PathOnHost:   String(testRootfs),
+						},
+					},
```

Comment:
Do we need to specify root drive here? Which handler requires this?

Reply:
I feel it would be best to keep them here just so that `ValidateLoadSnapshot()` can ensure that it's in place, but leaving it out still makes it pass, so you're technically correct. I would like to leave this here because leaving out the drives would not allow the user to interact with it until it adds said drives. What do you think?